main/.vuepress/themeConfig/nav.js

```diff
@@ -15,6 +15,30 @@ module.exports = [
         ariaLabel: 'Agoric Dev Tools Menu',
         link: '/getting-started/',
       },
+        text: 'Before Using Agoric',
+        ariaLabel: 'Before Using Agoric Menu',
+        link: '/getting-started/before-using-agoric/',
+      },
+      {
+        text: 'Starting a Project',
+        ariaLabel: 'Starting a Project Menu',
+        link: '/getting-started/start-a-project/',
+      },
+      {
+        text: 'Development Cycle',
+        ariaLabel: 'Development Cycle Menu',
+        link: '/getting-started/development-cycle/',
+      },
+      {
+        text: 'ERTP Introduction',
+        ariaLabel: 'ERTP Introduction Menu',
+        link: '/getting-started/ertp-introduction/',
+      },
+      {
+        text: 'Agoric CLI Guide',
+        ariaLabel: 'Agoric CLI Guide Menu',
+        link: '/getting-started/agoric-cli-guide',
```

Comment:
404, I think you need the trailing /
```suggestion
        link: '/getting-started/agoric-cli-guide/',
```